numpy/_core/shape_base.py

```diff
@@ -605,9 +602,9 @@ def _concatenate_shapes(shapes, axis):
     if any(shape[:axis] != first_shape_pre or
            shape[axis+1:] != first_shape_post for shape in shapes):
         raise ValueError(
-            'Mismatched array shapes in block along axis {}.'.format(axis))
+            f'Mismatched array shapes in block along axis {axis}.')
 
-    shape = (first_shape_pre + (sum(shape_at_axis),) + first_shape[axis+1:])
+    shape = first_shape_pre + (sum(shape_at_axis),) + first_shape[axis+1:]
```

Comment:
this is an unrelated change but it looks correct to me, the parens aren't necessary